analysis.py

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/env python3
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2023 IBM
+# Author: Misbah Anjum N <misanjum@linux.vnet.ibm.com>
+
+
+'''
+    analysis.py script is aimed to help in analysis/comparison of avocado test runs 
+    by generating a simple excel file (.xlsx). The results.json file which gets created 
+    after avocado run is passed as input in command line while running this script and 
+    depending on the flag/options provided, the excel analysis/omparison sheet will be 
+    generated.
+
+    Prerequsites:-
+    pip3 install pandas[excel]
```

Comment:
how about handling this in script itself

Reply:
The reason I did not handle this in the script was because there might be different pandas versions used by people. Some might have distro pandas installed instead. Handling this in the script itself might lead to problems with python pointing to multiple library versions., and thus errors.

Hence I added this in the file:-

```
    Prerequsites:-
    pip3 install pandas[excel]
    (or)
    dnf install python3-pandas python3-numpy python3-openpyxl
```

Please do let me know what you think about this.
Thanks.

Reply:
sure , make sense